### ISSUE TITLE
[Designer] [Bug 40018799](https://microsoft.visualstudio.com/OS/_workitems/edit/40018799): [Expense report.json-> Category-Amount Report]: Screen reader is not announcing proper information for "Air Travel Expense $300 expand" button control.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7614]
### Problem Description

[Bug 40018799](https://microsoft.visualstudio.com/OS/_workitems/edit/40018799): [Expense report.json-> Category-Amount Report]: Screen reader is not announcing proper information for "Air Travel Expense $300 expand" button control.

Test Environment:

Test Environment:
OS: Win11 (25136.1000)
UWP App version: 3.0.2206.2001
Screen reader: Narrator

Repro steps:

1. Launch the Adaptive Card application and Enable Narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to ExpenseReport.Json button by using Tab key and activate it.

4. Expense approval adaptive card will be appeared on right sight of the pane.

5. Navigate to Expense approval card by using Tab key.

6. Navigate to the arrow button present besides 'Air Travel Expense $300' & listen the screen reader narration.

Actual Result:
Screen reader is not announcing the name defined for 'Expand' button. Screen reader is announcing "Button" only. 

Expected Result:
Screen should announce the complete name of 'Expand' button. Screen reader should announce "Air Travel Expense $300 expand button collapsed". 

User Experience:
Screen reader users would be impacted if Screen reader is not announcing proper information for "Air Travel Expense $300 expand" button control. 

Note: This issue is repro for "Auto Mobile Expense $100 expand button collapsed " and "Excess Baggage Cost $50 expand button collapsed".

### Card JSON

```JSON
Designer Bug
```

**Comment by aaagrawal** [comment]
_Linked to Azure Boards Test Plans_
[### **Design**-Test Case: 28646792 Navigate to "ExpenseReportwithLabels.JSON" button](https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/28646792)

**Comment by vagpt** [comment]
Closing this issue as of now this issue is tracking with #8973
